src/baklava/CustomNode.vue

```diff
@@ -2,7 +2,10 @@
   <div
     :id="data.id"
     :class="classes"
-    :style="styles"
+    :style="[styles, shading]"
+    :title="showErrorMessages ? messages : undefined"
+    @mouseover="setShowErrorMessages(true)"
+    @mouseleave="setShowErrorMessages(false)"
```

Comment:
Do we need those `@mouseover` and `@mouseleave`? I thought the `title` prop would show it only when your mouse is over it right?

Reply:
This was @gidonkatten who advised me so maybe not? But i'd like to leave them in order to make a custom component/div in the future instead of the cheap `title`

Reply:
Agreed, that makes sense then!

Reply:
The mouse events are used for surrounding border so is needed

Reply:
@lancelotblanchard Oh I think you are right, my bad